### PR TITLE
feat(build) remove export from build script

### PR DESCRIPTION
**Pull request by NewEraCracker**
this moves the determination of the git commit to webpack-shared-config

LIB_JITSI_MEET_COMMIT_HASH is still kept for backward compatibility

related to #1951 and #1952

### CODE REVIEW

**Review by saghul — APPROVED** [review]
LGTM

**Comment by NewEraCracker** [comment]
I have pushed a change to set development version in case the git command fails (missing .git directory). I am producing this on Linux but I am fairly confident this works on windows. 

Maybe @garyhuntddn can also take a look to confirm this works.

Thanks!

File: webpack-shared-config.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
 /* global __dirname */
 
 const process = require('process');
+const commitHash = process.env.LIB_JITSI_MEET_COMMIT_HASH
+    || require('child_process')
```
Comment (saghul) [inline webpack-shared-config.js:5]: Can you do the require unconditional please?

**Review by saghul — COMMENTED** [review]
Left a comment I hadn't noticed last time around

**Review by saghul — APPROVED** [review]

**Comment by garyhuntddn** [comment]
Isn’t that comment something clever that the IDEs use to know that __dirname will be defined?

**Review by saghul — DISMISSED** [review]
LGTM. @damencho ?

**Comment by NewEraCracker** [comment]
I have rebased this on top of the most recent commit of master which includes the build system changes.

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
Jenkins please test this please.

**Comment by saghul** [comment]
Sorry for the delay @NewEraCracker !

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
Jenkins please test this please.

**Comment by jack5432109** [comment]
Looks like these changes are still relevant on windows. They fixed issues I saw while building on windows

**Comment by saghul** [comment]
Rebased at https://github.com/jitsi/lib-jitsi-meet/pull/2483

**Comment by saghul** [comment]
Landed in https://github.com/jitsi/lib-jitsi-meet/pull/2483

**Pull request closed without merging**